main.md

```diff
@@ -189,7 +189,12 @@ This draft defines a set of common data elements that are designed to be usable
 `datatypes`:
 :   An array of strings representing the kinds of data being requested from the resource. 
 
-If `datatypes` and `actions` are used in combination, the permissions the client requests is the cartesian product of both arrays. In the following example
+`identifier`:
+:   A string identifier indicating a specific resource available at the API. 
+
+An API MAY define its own extensions, subject to the `type` of the request. It is assumed that the full structure of each of the authorization data elements is tailored to the needs of a certain application, API, or resource type. The example structures shown above are based on certain kinds of APIs that can be found in the Open Banking space. 
+
+When different element types are used in combination, the permissions the client requests is the cartesian product of the values. In the following example
```

Comment:
I think this statement is too broad. I understand the way it is applied to locations, actions, and datatypes. 
I don't think it's applicable to type and I don't understand what the expected behavior would be for identifier.

Reply:
I see `type` as a different kind of field from the rest because it determines what else goes in the data object. It's function switching more than it is defining the access. So maybe it needs to be defined separately from the rest of the list? 

For `identifier` it's still part of the cartesian product: it's that identifier across all the different actions and locations and datatypes and everything else. You're saying I want read and write to this one account.